### ISSUE TITLE
translateExtent for ZoomableGroup -- map element jumps when clicked

### ISSUE THREAD

**Issue by TJBachorz** [zcreativelabs/react-simple-maps#235]
Can you please describe how the translateExtent prop for ZoomableGroup works?  

`                <ZoomableGroup 
                    center={[-150, 55]} 
                    minZoom={1} 
                    maxZoom={1} 
                    translateExtent={[[-150, 55], [-150, 55]]}
                >`

When I click on the space outside of the map element, the map element jumps a short distance away, usually cutting off part of the map element.  I'm attempting to make it so that my map elements are not movable or zoomable.  I'm only using the ZoomableGroup component for centering.

**Comment by mmartinsky** [comment]
I wonder if it's supposed to be `a2` in https://github.com/zcreativelabs/react-simple-maps/blob/master/src/components/useZoomPan.js#L76

```
const zoom = d3Zoom()
      .filter(filterFunc)
      .scaleExtent([minZoom, maxZoom])
      .translateExtent([[a1, a1], [b1, b2]])
      .on("start", handleZoomStart)
      .on("zoom", handleZoom)
      .on("end", handleZoomEnd)
```

**Comment by CheerfulMushroom** [comment]
Property "translateExtent " is used to define boundaries user can not go beyond.
translateExtent = [ [ minX, minY ], [ maxX, maxY ] ], where 
minY - upper bound in pixels
maxY - bottom bound in pixels
minX - left bound in pixels
maxX - right bound in pixels

The boundaries you have set are weird because you basically made your boundaries a single point
Try setting more appropriate boundaries (e.g. [ [ -150, -55 ], [ 150, 55 ] ]) and **make sure** that your map is able to **fit in the given boundaries**. 

Example of usage:
https://github.com/CheerfulMushroom/KTS_Frontend_2020_inFrontOfYou/blob/layout-fix-history-compare/src/pages/Home/components/mapChart.js#L69

If you want to test it, run it from branch "layout-fix-history-compare"

**Comment by zimrick** [comment]
#233 has been included in v2.2.0. This should fix some of the more confusing behaviour around translateExtent.
